.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
I think I know where this file is coming from. Probably auto generated by a Vscode plugin.Totally nothing wrong for pushing this of course, but when working on a large team, where not everyone is using vscode, this might not be a good file to push to the project codebase.

So in order for us to have a very close production-like experience, I recommend deleting this file, and adding it to the `.gitignore` file. That way, you use your local settings, while the rest of the team (even though you're alone in this case haha) can use theirs depending on their computers, or editors or settings.